cms/djangoapps/contentstore/views/item.py

```diff
@@ -295,6 +295,7 @@ def xblock_view_handler(request, usage_key_string, view_name):
         elif view_name in PREVIEW_VIEWS + container_views:
             is_pages_view = view_name == STUDENT_VIEW   # Only the "Pages" view uses student view in Studio
             can_edit = has_studio_write_access(request.user, usage_key.course_key)
+            can_edit_visibility = not view_name == 'container_child_preview'
```

Comment:
@cahrens I renamed the view in a5f157ffba77e69e1c20508c1e32a6832c6ee4e0.

f0ceab200bc165c6d39ceb545dcb1ca4f5d70010 implements an alternative approach that explicitly passes `can_edit_visibility` info from the library JS code instead of relying on knowledge about view names inside `xblock_view_handler`.